src/test/java/seedu/address/logic/parser/EditPersonCommandParserTest.java

```diff
@@ -36,31 +36,31 @@
 
```

Comment:
I like that the Tests were changed to accommodate the EditPersonCommand change. However, should we have another class to test the newly implemented EditEventCommandParser?